src/schema/article-renderer.tsx

```diff
@@ -120,15 +120,25 @@ export const articleColors = {
 }
 
 // TODO: needs type declaration
+
+interface RenderLeafProps {
+  leaf: {
+    color?: 'blue' | 'green' | 'orange'
+    em?: boolean
+    strong?: boolean
+  }
+  attributes: any
+  children: any
```

Comment:
React children should be `React.ReactNode` usually

Reply:
what about `attributes`? this comes up more often.

Reply:
Not sure, seems do be David's stuff (which is insipired by Slate's code).